keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -487,10 +489,9 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
 
         test("can retrieve the manifest for the resources") {
           val resource = deliveryConfig.resources.random()
-
-          getDeliveryConfig(resource)
-            .isSuccess()
-            .isEqualTo(deliveryConfig)
+          val persistedDeliveryConfig = repository.deliveryConfigFor(resource.id)
+          expectThat(persistedDeliveryConfig)
+            .isEqualTo(deliveryConfig.copy(metadata = persistedDeliveryConfig.metadata))
```

Comment:
Is this really proving anything?

Reply:
This was me just basically trying to keep a check equivalent to what was there before (there was already an `isEqualTo` but the adding of `createdAt` to the metadata when reading makes the objects different only for that reason).

Reply:
so `expectThat(persistedDeliveryConfig.metadata["createdAt"]).isNotNull()`?

Reply:
No, this test was unrelated to my change. It was just checking that you could retrieve a delivery config for a resource id. But I had to fix it after my change because the metadata wouldn't match. I changed the test a bit and added a comment, hopefully it's clearer now.